jest.config.js

```diff
@@ -107,15 +107,15 @@ module.exports = {
   // rootDir: null,
 
   // A list of paths to directories that Jest should use to search for files in
-  // roots: [
-  //   "<rootDir>"
-  // ],
+  roots: [
+    "<rootDir>/test/unit"
+  ],
 
   // Allows you to use a custom runner instead of Jest's default test runner
   // runner: "jest-runner",
 
   // The paths to modules that run some code to configure or set up the testing environment before each test
-  // setupFiles: [],
+  setupFiles: ['<rootDir>/test/setupTests.js'],
```

Comment:
The Jest and Enzyme updates now require us to specify a React adapter to use - so we create this `setupTests.js` setup file to run before our tests.